install_influxdb.sh

```diff
@@ -0,0 +1,30 @@
+#/bin/bash
+set -e
+
+# Usage
+if [ -z "$1" ]; then
+    printf "\nInfluxDB version not provided.\n"
+    printf "usage: %s <INFLUXDB VERSION>\n" "$0"
+    exit 1
+fi
+
+# Download
+FILEURL="http://s3.amazonaws.com/influxdb/influxdb_$1_amd64.deb"
+TEMPFILE=$(mktemp --suffix=.deb)
+printf "Trying to download $FILEURL, saving as $TEMPFILE\n"
+curl --fail --progress-bar -L "http://s3.amazonaws.com/influxdb/influxdb_$1_amd64.deb" -o "$TEMPFILE"
+
+# Install & start
+printf "Installing InfluxDB\n"
+dpkg -i $TEMPFILE
+service influxdb start
+
+# Create initial db
+for i in 1 2 3 4 5 6; do
```

Comment:
while this will probably not fail frequently, it might provide some sporadic failure due to machines being slow.
You might want to check if the port is open instead? Not a must, but if we stumble upon this once, we should fix it.